### PR TITLE
Feat: Add worker id to container list when token is cluster admin

### PR DESCRIPTION
**Pull request by nickpetrovic**
Resolve BE-2021

### CODE REVIEW

File: pkg/gateway/services/container.go [hunk 1]
```diff
@@ -13,26 +13,58 @@ import (
...
-			Ok:       false,
-			ErrorMsg: "Unable to list containers",
-		}, nil
+	type containerDetails struct {
+		WorkerId  string
+		MachineId string
 	}
 
-	containers := make([]*pb.Container, 0, len(containerStates))
+	var (
```
Comment (luke-lombardi) [inline pkg/gateway/services/container.go:21]: Could we move this admin specific stuff out to another helper function or something?

**Review by luke-lombardi — CHANGES_REQUESTED** [review]

File: pkg/gateway/services/container.go (hunk 1 above)
Comment (nickpetrovic) [reply pkg/gateway/services/container.go:17]: We can. Just updated see if it makes better sense.

**Review by dleviminzi — APPROVED** [review]

File: pkg/gateway/services/container.go (hunk 1 above)
Comment (luke-lombardi) [reply pkg/gateway/services/container.go:17]: Yeah I think this is a little cleaner

**Review by luke-lombardi — APPROVED** [review]

File: sdk/pyproject.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "beta9"
-version = "0.1.100"
+version = "0.1.101"
```
Comment (luke-lombardi) [inline sdk/pyproject.toml:3]: I think we just release 101, need a bump here

**Review by luke-lombardi — CHANGES_REQUESTED** [review]

File: sdk/pyproject.toml (hunk 2 above)
Comment (nickpetrovic) [reply sdk/pyproject.toml]: fixed

Comment (dleviminzi) [reply sdk/pyproject.toml]: just did 102, my bad

**Pull request merged**